packages/design-system-docs/src/pages/components/Table/Table.example.jsx

```diff
@@ -0,0 +1,95 @@
+import {
+  Table,
+  TableBody,
+  TableCaption,
+  TableCell,
+  TableHead,
+  TableHeader,
```

Comment:
I noticed that `TableHeader` and `TableHead` are similar in name. I know they correspond to `thead` and `th`, which are just similar to begin with, but I wonder if we can clarify the naming?

Maybe `th` and `td` could correspond to `TableHeaderCell` and `TableDataCell`? This way we can differentiate cells from the table caption, header, and body